krun/platform.py

```diff
@@ -26,13 +26,11 @@
 class BasePlatform(object):
     __metaclass__ = ABCMeta
 
-    # Each temperature sensor must cool to its starting temperature plus
-    # TEMP_THRESHOLD_DEGREES. An absolute value in degrees centigrade.
-    TEMP_THRESHOLD_DEGREES = 1
+    TEMP_THRESHOLD_DEGREES = 3
 
     TEMP_MANDATORY_WAIT = 1
     TEMP_POLL_FREQ = 10                 # seconds between polls
-    TEMP_POLLS_BEFORE_MELTDOWN = 60     # times 10 = ten mins
+    TEMP_POLLS_BEFORE_MELTDOWN = 360    # times 10 = 1 hour
```

Comment:
I ask because, we can't do `time.sleep(ONE_HOUR)`. We need to check every X seconds up to an hour. If we check every second, we are grand as long as the check itself being so frequent won't prevent us from cooling down.